README.md

```diff
@@ -1,23 +1,29 @@
-# Yomichan
+# Yomitan
 
-*Note: this project is no longer maintained. Please see [this
-post](https://foosoft.net/posts/sunsetting-the-yomichan-project/) for more information.*
+> :wave: **This project is a community fork of yomichan** (which was [sunset](https://foosoft.net/posts/sunsetting-the-yomichan-project/) by its owner on Feb 26 2023).
+>
+> The primary goal is to **keep the project alive and working on latest browser versions and doing bugfixes**. (Feature improvements are considered secondary.)
+> 
+> Since the owner requested forks be uniquely named, we have chosen a new name: *yomitan*. (*-tan* is an honorific used for anthropomorphic moe characters.) While we've renamed the codebase and made judicious use of find-and-replace, it is entirely the hard work of foosoft and numerous other open source contributors from 2016-2023 and we do not claim any credit.
```

Comment:
```suggestion
> Since the owner requested forks be uniquely named, we have chosen a new name: *yomitan*. (*-tan* is an honorific used for anthropomorphic moe characters.) While we've renamed the codebase and made judicious use of find-and-replace, it is entirely the hard work of [FooSoft](https://github.com/FooSoft) and numerous other open source contributors from 2016-2023 and we do not claim any credit.
```